.travis.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - &base-test
       stage: test
       go_import_path: github.com/redhat-developer/odo
-      go: "1.11"
+      go: "1.11.2"
```

Comment:
We need to also update the go version in deploy stage here https://github.com/redhat-developer/odo/blob/5622eb92ba3ece4d42ddd8c4bb00cf6f0b8c8ef0/.travis.yml#L109